src/api/tokens.ts

```diff
@@ -1,16 +1,22 @@
 // {"multi_use": true, "valid_for": "10 days"}
 
-import { RPCS, supabaseClient } from 'services/supabase';
+import { PostgrestSingleResponse } from '@supabase/postgrest-js';
+import { RPCS, supabaseClient, supabaseRetry } from 'services/supabase';
 
 export const createRefreshToken = async (
     multi_use: boolean,
     valid_for: string
 ) => {
-    return supabaseClient
-        .rpc<{ id: string; secret: string }>(RPCS.CREATE_REFRESH_TOKEN, {
-            multi_use,
-            valid_for,
-        })
-        .throwOnError()
-        .single();
+    return supabaseRetry<
+        PostgrestSingleResponse<{ id: string; secret: string }>
+    >(
+        () =>
+            supabaseClient
+                .rpc(RPCS.CREATE_REFRESH_TOKEN, {
+                    multi_use,
```

Comment:
Where do you stand on using snake case for the input parameters of API-related functions?

Reply:
I don't really prefer snake case unless it is for something that is "from the back end". So here it kinda makes sense so that we don't have to translate the data.